build.gradle

```diff
@@ -27,6 +27,7 @@ ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
 ext.slf4jVersion = "1.7.30"
 ext.axiomVersion = "1.2.22"
 ext.quartzVersion = project.quartzVersion
+ext.stdlibTimeVersion = project.stdlibTimeVersion
```

Comment:
We don't need this. (All the `ext.*` definitions in that matter. We can move the slf4j and axiom versions to the `gradle.properties` file too.

Reply:
What I meant was, we can remove all the `ext.*` definitions here. We don't need them.